dotnet/src/SemanticKernel.Abstractions/Models/PromptModel.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.AI;
+
+namespace Microsoft.SemanticKernel.Models;
+
+/// <summary>
+/// Prompt model files.
+/// </summary>
+public sealed class PromptModel
```

Comment:
I would change it to have at least `Function` in the name. Not sure if `SemanticFunction` will still be there, if the name is `PromptFunction` then I would suggest
`PromptFunctionModel` or `PromptFunctionConfig` (some variables use config in the name, like semanticFunctionConfig